config.yaml

```diff
@@ -0,0 +1,5 @@
+templateDir: /local/templates
+files:
+  api_paginated_listers.mustache:
+    templateType: API
```

Comment:
Was curious what the options are:
https://openapi-generator.tech/docs/customization#:~:text=The-,templateType%20option,-will%20default%20to